hawc/templates/base.html

```diff
@@ -107,6 +107,9 @@
     <script type="text/javascript" src="{% static 'vendor/jqueryui/1.10.3/jquery-ui.min.js' %}"></script>
     <script type="text/javascript" src="{% static 'vendor/htmx/1.6.1/htmx.min.js' %}"></script>
     {% render_bundle 'main' %}
+    {% if form %}
```

Comment:
this is very nice. we do have to watch out to make sure that we still need to add `form.media` on any htmx stuff, but that makes sense conceptually so i like this pattern